packages/core/config/webpack.common.js

```diff
@@ -340,6 +340,11 @@ module.exports = (webpackEnv) => {
                 filename: 'static/media/[name].[contenthash:8][ext]',
               },
             },
+            // (8)
+            !redwoodConfig.experimental.realtime.enabled && {
```

Comment:
Will have to rethink of to toggle. Late PR's will introduce a `graphql` toml config section, so could put there as `realtime` or even useSSE?